### ISSUE TITLE
Pre match graph data

### ISSUE THREAD

**Issue by AarushPSchool** [sjcbulldog/xeroscout#36]
Can you add average notes for auto, teleop and amp and add the ability to filter av auto notes by starting possition for all teams (pre match graphs).

**Comment by sjcbulldog** [comment]
Aarush, per the email I sent, adding something specific for  this year does not make sense.  Have you looked at the expression capabilities for the pre-match graphs?

**Comment by AarushPSchool** [comment]
Yes, I managed to add everything except the filter by starting pos for auto, (I want to be able to look at certain tag(s) and say, based on the dropdown position, what the average value was (for like auto game pieces and all the center note bools).
